### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by hugopassos**
I've been doing the commits on master branch all the time in this project and I just realized this mistake now. I decided not to remove the files on master to keep track of my commits.

### CODE REVIEW

**Comment by stratospherique** [comment]
Hi @hugopassos :hand: 
Awesome work on the project. However, I think you should consider fix the following issues:

- Your navigation bar is missing the current date, sign-in and subscribe links.
- try to leave some of the content (e.g. sign-in and subscribe) for page width below the 900px (not 767px).
- Try to use links instead of plain text, it makes the page more vibrant and engaging for the visitors.
- Run your HTML through the W3C validator and fix the raised issues.
- For the next project, try to follow the [Git Flow guidelines](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59882084/)

**Comment by stratospherique** [comment]
### Pull Request Under the review By Ahmed Mahfoudh (TSE)

**Comment by stratospherique** [comment]
> ### Pull Request Under the review By Ahmed Mahfoudh (TSE)

You are close but a small changes are required, please take a look at my comments and re-submit for review :+1:

- Your navigation bar is missing the current date, sign-in and subscribe links.
- In the Navigation bar, on medium screen, the menu should transform to a hamburger and current date should disappear.
- [OPTIONAL] Try to use links instead of plain text, it makes the page more vibrant and engaging for the visitors.

**Review by stratospherique — CHANGES_REQUESTED** [review]
> ### Pull Request Under the review By Ahmed Mahfoudh (TSE)

You are close but a small changes are required, please take a look at my comments and re-submit for review :+1:

- Your navigation bar is missing the current date, sign-in and subscribe links.
- In the Navigation bar, on medium screen, the menu should transform to a hamburger and current date should disappear.
- [OPTIONAL] Try to use links instead of plain text, it makes the page more vibrant and engaging for the visitors.

File: index.html
Comment (bolah2009) [inline index.html:17]: #### `AWESOME`

Nice use of the `header` element for introductory content of the page. :stars:

File: index.html
Comment (bolah2009) [inline index.html:49]: #### `HINT`

Are we using the `figure` element effectively this way? :thinking: Consider using `<figcaption>` element instead of the `p` element here. Alternatively, you can also wrap the `<figcaption>` with the `p` element. See example below:
```
   <figure>
       <img src="../newsweek-momentum-awards-2019.png" alt="Momentum Awards">
       <figcaption>  
            <p class="img-absolute-p" class="font-weight-bold">ANALYSIS</p>
      </figcaption>
   </figure>
```

Check [here](https://html.spec.whatwg.org/multipage/grouping-content.html#the-figcaption-element) for more details on using `<figcaption>`.

File: index.html
Comment (bolah2009) [inline index.html:56]: See the comment on using `<figcaption>`.

File: index.html
Comment (bolah2009) [inline index.html:64]: See the comment on using `<figcaption>`.

File: index.html
Comment (bolah2009) [inline index.html:72]: See the comment on using `<figcaption>`.

File: index.html
Comment (bolah2009) [inline index.html:83]: See the comment on using `<figcaption>`.

File: index.html
Comment (bolah2009) [inline index.html:45]: #### `REQUIRED`

Use `<main>` element in L41 instead: ` <div class="main-wrapper container-fluid">`

The `div` is classified as a `main-wrapper` using class attribute. Using `main` adds more semantic meaning making it represent the **`dominant content of the <body> of a document.`**

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Nice Job @hugopassos :clap: Almost there :smile: 

The [clone](https://raw.githack.com/hugopassos/Newsweek-Bootstrap/Home-Page/index.html) looks nice, responsive and uses bootstrap class.

Before giving a :+1: let's make some **required** changes.
- Add appropriate linter file. [See comment below](https://github.com/hugopassos/Newsweek-Bootstrap/pull/1#pullrequestreview-276952374)
   > [Set up a code linter](https://github.com/microverseinc/linters-config/tree/master/css).
- Use semantic elements. Think of where semantic elements would have made the page more accessible. You can inspect [source page](https://www.newsweek.com/) for hints, see comments and link below.
   > Remember to use [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp).
- [Next Project] Use GitHub Flow: For HTML & CSS projects, use only one feature branch (e.g. `embedding-images-and-video`, `html-forms`, `responsive-design`, etc) and **do not merge any feature branch to master until it's been approved**.
   > Follow the [Github Flow guidelines](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/).
- [OPTIONAL] Use more bootstrap classes

Go through the [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540308/) and [odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/using-bootstrap) project requirements for more details.


Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Review by bolah2009 — COMMENTED** [review]
#### `REQUIRED`

Add `.stickler.yml` file to the root of the project. Also let's use [this file](https://github.com/microverseinc/linters-config/blob/master/css/.stickler.yml). :point_down: 

``` yml
linters:
  stylelint:
    config: 'stylelint.config.js'
```
- Also, add the `stylelint.config.js` that was referenced above, this can be found [here](https://github.com/microverseinc/linters-config/blob/master/css/stylelint.config.js)

If you are having an issue with setting up linter, read the `Troubleshooting` section of [this repo](https://github.com/microverseinc/linters-config/tree/master/css).

**Pull request closed without merging**
